projects/grafana/grafana-ocp.yaml

```diff
@@ -9,7 +9,38 @@ metadata:
       Grafana server with patched Prometheus datasource.
     iconClass: icon-cogs
     tags: "metrics,monitoring,grafana,prometheus"
+parameters:
```

Comment:
based on that yaml we'll deploy the grafana behind oauth
@stevekuznetsov @kargakis